subiquity/server/controllers/network.py

```diff
@@ -246,8 +246,7 @@ async def apply_autoinstall_config(self, context):
             self.apply_config(context)
         with context.child("wait_for_apply"):
             await self.apply_config_task.wait()
-        self.model.has_network = bool(
-            self.network_event_receiver.default_routes)
+        self.model.has_network = self.network_event_receiver.has_default_route
```

Comment:
100% nothing to do with this branch but I think this line is pointless given that configured does the same and is called immediately after apply_autoinstall_config?